api/server/server.go

```diff
@@ -5,11 +5,33 @@ import (
 	"log"
 	"net"
 
+	"fmt"
 	"github.com/appcelerator/amp/api/rpc/project"
 	"github.com/appcelerator/amp/api/rpc/service"
+	"github.com/appcelerator/amp/data/elasticsearch"
 	"google.golang.org/grpc"
+	"gopkg.in/olivere/elastic.v3"
+	"os"
 )
 
+var (
+	es elasticsearch.ElasticSearch
+)
+
+func init() {
+	// Get elasticsearch url from environment
+	ES_URL := os.Getenv("ES_URL")
```

Comment:
All caps implies a constant, which it is not. In any case, it isn't idiomatic in Go to use all caps even for constants.